### PR TITLE
bold usage report changes, fixes #70

### LINKED ISSUE #70: Usage Report in `create-cluster` highlights changed items
## Description
In https://github.com/arkime/aws-aio/issues/64, we added a usage report as part of `create-cluster` so users could get a sense of what would be provisioned, for business and operational planning purposes.  This report could be further refined by highlighting the items that have changed, if any, so they are more obvious

## Acceptance Criteria
* Usage report items that have changed are "highlighted", maybe with a different text color, an asterisk, etc.

### PR DESCRIPTION
**Pull request by awick**
We now bold changes in the usage report. When it isn't a change or the old value is None we just print the value.

Example:
image

## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: test_manage_arkime/core/test_usage_report.py [hunk 1]
```diff
@@ -30,21 +30,21 @@ def test_WHEN_UsageReport_get_report_THEN_as_expected():
     # Check the results
     expected_report = (
```
Comment (chelma) [inline test_manage_arkime/core/test_usage_report.py:31]: Can we please add either a new test or update a line in the existing test to show the difference in behavior when we have `None` as the old value and when we don't?

**Review by chelma — CHANGES_REQUESTED** [review]
Great change!  Would like a few minor tweaks, though.

**Review by chelma — APPROVED** [review]
LGTM - thanks for the change!
